[DE] Add `Rules/Code of conduct for mapping and modding` (#13904)
ppy/osu-wiki: wiki/Rules/Code_of_conduct_for_modding_and_mapping/de.md

# Verhaltenskodex für Modding und Mapping

> Der **Verhaltenskodex für Modding und Mapping** beinhaltet Regeln und Richtlinien, die für das gesamte [Modding](/wiki/Modding)- und [Mapping](/wiki/Beatmapping)-Umfeld gelten, und ist eine Erweiterung der [osu! Community-Regeln](/wiki/Rules). Um sicherzustellen, dass Mapping- und Modding-Diskussionen in einem konstruktiven, positiven und produktiven Umgang stattfinden und dass für alle dieselben Spielregeln gelten, ist ein Verhaltenskodex unerlässlich. Die Einhaltung dieses Verhaltenskodex ist bei der Beteiligung an Mapping- und Modding-Diskussionen verpflichtend. Verstöße gegen diese Regeln führen möglicherweise zu [Strafen](/wiki/Silence).

Reviewer: ```suggestion
Der **Verhaltenskodex für Modding und Mapping** beinhaltet Regeln und Richtlinien, die für das gesamte [Modding](/wiki/Modding)- und [Mapping](/wiki/Beatmapping)-Umfeld gelten, und ist eine Erweiterung der [osu! Community-Regeln](/wiki/Rules). Um sicherzustellen, dass Mapping- und Modding-Diskussionen in einem konstruktiven, positiven und produktiven Umfeld stattfinden, ist ein Verhaltenskodex unerlässlich, um alle auf denselben Stand zu bringen. Die Einhaltung dieses Verhaltenskodex ist bei der Beteiligung an Mapping- und Modding-Diskussionen verpflichtend. Verstöße gegen diese Regeln führen möglicherweise zu [Strafen](/wiki/Silence).
```

In the English version, it says "to get everyone on the same page". I think "um alle auf denselben Stand zu bringen" conveys the intended meaning better here.

## Verhalten

- **Durch Modding soll Mappern geholfen werden, damit sie besser werden.** Wenn du kein Interesse an einem gepflegten Umgang hast oder Menschen nicht helfen möchtest, halte dich aus der Diskussion raus.
- **Beachte, was du postest, warum du es postest und wo du es postest.** Es ist wichtig, dass deine Intention deutlich wird und dein Beitrag leicht verständlich ist.
- **Sei aufgeschlossen.** An den Diskussionen können sich alle Nutzer beteiligen. Jeder ist eingeladen, teilzunehmen, auch die, die neu in der Community sind. Sei geduldig, zeige Verständnis und erkläre so viel wie nötig, damit jeder dazulernen und sich schneller verbessern kann.
- **Behandle andere wie du selbst behandelt werden möchtest.** Nette, hilfreiche Vorschläge kommen besser an als feindselige. Bleib beim Thema, vermeide persönliche Angriffe und verbreite keine unangebrachten Witze sowie Hasskommentare, damit die Diskussion nicht aus dem Ruder läuft.

> - **Nutze die [Diskussionsseite](/wiki/Beatmap_discussion).** Besprich deine Vorschläge mit anderen aus der Community. Ansonsten kann es passieren, dass deine Bedenken von niemanden gehört werden, der diese angemessen umsetzen kann.

Reviewer: ```suggestion
- **Nutze die [Diskussionsseite](/wiki/Beatmap_discussion).** Besprich deine Vorschläge mit anderen aus der Community. Ansonsten kann es passieren, dass deine Bedenken von niemandem gehört werden, der diese angemessen umsetzen kann.
```

- **Geh auf den Zusammenhang ein.** Wenn in deinen Antworten auf Mods alle relevanten Informationen enthalten sind, wird es für zukünftige Modder einfacher, die Diskussion zu verstehen.

## Weiterer Austausch und Diskussion

Manchmal ziehen sich Diskussionen in die Länge und ein einzelner Mod sowie eine Antwort darauf reichen nicht aus, um alle Teilnehmer zu einer Einigung zu bringen. Vielleicht ist der Mapper mit einem Vorschlag nicht einverstanden und der Modder ist nicht mit der Antwort zufrieden. Idealerweise sollten alle Diskussionen innerhalb eines angemessenen Zeitraums beendet werden.

Beachte die folgenden Punkte, wenn es zu keiner Einigung kommt:

- **Frag, wenn dir etwas unklar ist.** Frag die betreffende Person, damit sie dir erklärt, was sie meint. In der Regel sind Personen bereitwillig, dir Dinge mit mehr Details zu erklären.
- **Frag jemanden mit mehr Erfahrung, wie beispielsweise ein [BN](/wiki/People/Beatmap_Nominators)- oder [NAT](/wiki/People/Nomination_Assessment_Team)-Mitglied.** Obwohl die Teams viel zu tun haben, sollten sie dir helfen können, eine Einigung zu erzielen.

> - **Sei offen für andere Meinungen.** Wenn du merkst, dass eine passendere Alternative zu deinem ursprünglichen Vorschlag existiert, sei nicht gekränkt. Es gibt keinen falschen Meinungen, also versuche, andere Ansichten zu verstehen, anstatt sie abzusprechen.

Author: ```suggestion
- **Sei offen für andere Meinungen.** Wenn du merkst, dass eine passendere Alternative zu deinem ursprünglichen Vorschlag existiert, sei nicht gekränkt. Es gibt keine falschen Meinungen, also versuche, andere Ansichten zu verstehen, anstatt sie abzusprechen.
```
